### DISCUSSION TITLE
onClick event not working

### DISCUSSION THREAD

**Discussion by Toubef** [atomiks/tippyjs#1150] [Q&A]
Hi,

I'm trying to get tippy a button onClick event fired inside tippy html content.
For some reason I cannot not get this to work.

Here is my tippyJs config
`    returnTippy = tippy('#showTemperatureLocation',{
      content: info,
      placement: 'bottom',
      animation: 'shift-toward-extreme',
      //trigger: 'click',
      allowHTML: true,
      interactive: true,
      delay: [0, 1000],
      onShown: function() {
        $("#refreshWeather").off("click").on("click", function(){
          alert('Weather is refreshing');
        });
      }
    });`

Am I doing something wrong or why cannot I not get thar alert shown?
Please advice.

Thanks.
-Toube

**Comment by arladmin** [comment]
@Toubef Hi,

Were you able to implement this use-case?
